addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/internal/SeneyeHandlerFactory.java

```diff
@@ -28,7 +28,7 @@
  *
  * @author Niko Tanghe - Initial contribution
  */
-@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.seneye", configurationPolicy = ConfigurationPolicy.OPTIONAL)
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.seneye", configurationPolicy = ConfigurationPolicy.REQUIRE)

```

Comment:
Please see my other comment here. I was wrong, sorry. You can keep the default, i.e. just remove the `configurationPolicy`.

Reply:
Ok